style.css

```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  min-width: 300px;
+  max-width: 1300px;
+  color: black;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 1em;
+  margin: 0 auto !important;
+}
+
+nav {
+  width: 100%;
+}
+
+.navigation {
+  display: grid;
+  grid-template-columns: auto;
+  color: white;
+}
+
+.card {
+  border: none;
+}
+
+.border-primary {
+  color: #7c7c7c !important;
+}
+
+.first-grid a {
+  text-decoration: none;
+  color: white;
+}
+
+.second-grid a {
+  text-decoration: none;
+  color: black;
+}
+
+.newsweek a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world2 a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.anchor a {
+  color: black;
+}
+
+.anchor2 a {
+  text-decoration: none;
+  color: black;
+}
+
+.anchor3 a {
+  color: black;
+}
+
+.pandemic-section a {
+  text-decoration: none;
+  color: #7c7c7c;
+}
+
+.second-nav a {
+  color: black;
+  font-weight: bold;
+}
+
+.second-grid a:hover {
+  color: #f72210;
+}
+
+.first-grid {
+  background-color: #f72210;
+  font-size: 1.5rem;
+}
+
+.second-grid {
+  background-color: #fff;
+  font-size: 1.5rem;
+}
+
+.subscribe-nav {
+  background-color: #fff;
+  color: #f72210 !important;
+}
+
+.SandS {
+  position: relative;
+  top: 1em;
+  right: 1em;
+}
+
+.fa-search {
+  color: black;
+}
+
+.newsweek {
+  font-size: 2em;
+  color: #f72210;
+}
+
+.grey {
+  color: #a0a0a0;
+  font-size: 1.5rem;
+}
+
+.world {
+  position: relative;
+  background-color: #eef3f0;
+  top: -1em;
+  font-size: 1.5rem;
+  width: 7rem;
+  color: #f72210;
+}
+
+.world2 {
+  position: relative;
+  background-color: #eef3f0;
+  top: -1.9em;
+  font-size: 1.5rem;
+  width: 7rem;
+}
+
+.pandemic {
+  background-color: #c2d4d7;
+  position: relative;
+  bottom: 2.3em;
+  width: 6em;
+  font-size: 10px;
+}
+
+.featured {
+  position: relative;
+}
+
+.paragraph {
+  font-size: 0.9rem;
+  color: black;
+}
+
+.pandemic-section a:hover {
+  text-decoration: none;
+  color: #7c7c7c;
+}
+
+.latest-news-text {
+  position: absolute;
+  left: 11em;
+  top: 1em;
+}
+
+.subscribe-p {
+  font-size: 1.3rem;
+  position: absolute;
+  left: 15em;
+  bottom: 7em;
+}
+
+.pandemich2 {
+  font-size: 1.5em;
+}
+
+.opinion-img {
+  width: 90px;
+  height: 90px;
+}
+
+.fauzi {
+  border-top: 1px solid #7c7c7c;
+  border-bottom: 1px solid #7c7c7c;
+  padding-top: 1em;
+}
+
+.sponsored {
+  background-image: url("images/sponsored.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 300px;
+  border: 2px solid grey;
+}
+
+.sponsored-text {
+  padding-top: 300px;
+}
+
+.anchor {
+  color: black;
+  font-size: 1rem;
+}
+
+.anchor a:hover {
+  text-decoration: underline red;
+}
+
+.anchor2 {
+  color: black;
+  font-size: 0.9rem;
+}
+
+.anchor2 a:hover {
+  color: red;
+}
+
+.anchor3 {
+  color: black;
+  font-size: 0.9rem;
+}
```

Comment:
CSS styling is repeated for anchor2 and anchor3. You can read [this](https://speckyboy.com/good-bad-css-practices/) article as a reference. [Pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)